functests/1_performance/performance.go

```diff
@@ -211,6 +212,29 @@ var _ = Describe("[rfe_id:27368][performance]", func() {
 				}
 			}
 		})
+
+		It("[test_id:XXXX][crit:high][vendor:cnf-qe@redhat.com][level:acceptance] stalld daemon is not running on the host", func() {
+			By("Running on a kernel version lower than 4.18.0-240.22.*")
+			invalidStalldKernelVersion := "4.18.0-240.21.1.el8_3.x86_64"
+			randomWorkerNode := workerRTNodes[rand.Intn(len(workerRTNodes))]
+			tuned := tunedForNode(&randomWorkerNode)
+			customUnameCmd := []string{"sed", "-i", fmt.Sprintf("'$ a uname_string = %s'", invalidStalldKernelVersion), "/etc/tuned/tuned-main.conf"}
```

Comment:
if memory serves well this just appends the `uname_string = ...` to the file, right? If so could a simpler `echo "uname_string = ..." >> /etc/tuned/tuned-main.conf` work? (or anything equivalent)